app/services/meeting_starter.rb

```diff
@@ -31,11 +38,13 @@ def call
 
   private
 
-  def computed_options
+  def computed_options(access_code:)
     room_url = File.join(@base_url, '/rooms/', @room.friendly_id, '/join')
+    moderator_message = "#{I18n.t('meeting.moderator_message')}<br>#{room_url}"
+    moderator_message += "<br>#{I18n.t('meeting.access_code', code: access_code)}" if access_code.present?
     {
       # TODO: - ahmad: Find a way to localize
```

Comment:
Shouldn't the todo comment be removed now?

Reply:
Yep